libraries/android/settings.gradle

```diff
@@ -1,5 +1,5 @@
 pluginManagement {
-    gradle.ext.agpVersion = '7.2.1'
+    gradle.ext.agpVersion = '7.4.2'
```

Comment:
If we use `includeBuild` with `WCAndroid`, it means that we'll also have to bump AGP in `WCAndroid` to `7.4.2`. Then, we'll have to bump `FluxC` to `7.4.2` (if we want to keep included builds), then WPAndroid 😞 . It creates chain of work dependent from the authors of React Native Gradle Plugin - I feel it might be dangerous in a long run. WDYT?

Reply:
I am working on updating JDK and AGP for all projects, so hopefully it won't become a big issue. Unfortunately, if we don't update AGP, the project did not build for me. So, I felt it was a must at the time. However, we can certainly try again to see if we can get `7.2.1` to work.

Reply:
Gotcha. At the time of writing my first comment, I don't know why, but I assumed that React Native Gradle Plugin has not fixed version, but it's always somehow the newest one (like "+" dependency). That's of course incorrect - the React Native Gradle Plugin is bundled with the React Native version we define in `ext`.

That's where my concerns with AGP synchronization raised.

For now, at this stage of experiment, I think it's completely fine to use AGP 7.4.2 here. I've updated AGP to 7.4.2 as well on https://github.com/woocommerce/woocommerce-android/pull/9225 PR so we can still include `WooCommerce-Shared` to `WCAndroid`.